dll/win32/kernel32/client/sysinfo.c

```diff
@@ -413,6 +413,30 @@ GetNumaAvailableMemoryNode(IN UCHAR Node,
     return TRUE;
 }
 
+/*
+ * @implemented
+ */
+BOOL
+WINAPI
+GetPhysicallyInstalledSystemMemory(
+    _Out_ PULONGLONG TotalMemoryInKilobytes)
+{
+    MEMORYSTATUSEX status;
+
+    if (TotalMemoryInKilobytes == NULL)
+    {
+        SetLastError(ERROR_INVALID_PARAMETER);
+        return FALSE;
+    }
+
+    status.dwLength = sizeof(status);
+    if (!GlobalMemoryStatusEx(&status))
```

Comment:
> The GetPhysicallyInstalledSystemMemory function retrieves the amount of physically installed RAM from the computer's SMBIOS firmware tables. This can differ from the amount reported by the [GlobalMemoryStatusEx](https://learn.microsoft.com/en-us/windows/desktop/api/sysinfoapi/nf-sysinfoapi-globalmemorystatusex) function, which sets the ullTotalPhys member of the [MEMORYSTATUSEX](https://learn.microsoft.com/en-us/windows/desktop/api/sysinfoapi/ns-sysinfoapi-memorystatusex) structure to the amount of physical memory that is available for the operating system to use.

If this silently does do that, it will be very hard to track bugs down stemming from this.
I'd like to see at least a debug print (DPRINT1?) here indicating that a workaround is being used